introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vehicula feugiat tortor, non tempus ligula varius nec. Pellentesque faucibus ligula dui, vel hendrerit odio aliquam sed. Mauris volutpat odio ac lectus ultricies euismod. Nunc gravida placerat mattis. Vivamus auctor lobortis nulla, a rhoncus neque mollis vel. Duis lorem nunc, porttitor tempus justo a, eleifend fringilla mauris. Quisque in lacinia mauris. Donec commodo dui metus, ac semper lectus molestie quis. Cras nec ornare dui. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.